app/src/main/java/com/internship/move/ui/main/view/authentication/ClickCallback.kt

```diff
@@ -0,0 +1,6 @@
+package com.internship.move.ui.main.view.authentication
+
+fun interface ClickCallback {
+
+    fun invoke()
```

Comment:
I believe you could mark this as the operator itself `operator fun invoke()`
This way you could call `onClickCallback()` if ever you instantiate it as a variable. This is because of the `operator` modifier